ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -138,6 +142,26 @@ export class Mint extends IronfishCommand {
       fee = CurrencyUtils.decodeIron(input)
     }
 
+    if (!flags.confirm) {
+      const nameString = name ? `Name: ${name}` : ''
+      const metadataString = metadata ? `Metadata: ${metadata}` : ''
+      const includeTicker = !!assetId
+      const amountString = CurrencyUtils.renderIron(amount, includeTicker, assetId)
+      const feeString = CurrencyUtils.renderIron(fee, true)
+      this.log(`
+You are about to mint ${nameString} ${metadataString}
+${amountString} plus a transaction fee of ${feeString} with the account ${account}
+
+* This action is NOT reversible *
+`)
```

Comment:
This string got pretty awkward to craft since everything is optional - it would be great to refactor this to prep a raw transaction before confirming, so we could parse out the asset id, name, etc always